service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java

```diff
@@ -96,23 +97,23 @@ public static RoboticsData invalidRoboticsDataWithZeroOwningCourt() {
                 .build())
             .relationshipToChild("Uncle")
             .dob("2-MAY-1944")
-            .confidential(true)
+            .confidential(false)
             .build());
     }
 
     public static Set<Child> expectedChildren() {
         return ImmutableSet.of(Child.builder()
-            .firstName("Tom")
-            .lastName("Reeves")
-            .gender("MALE")
-            .dob("15-JUN-2018")
-            .isParty(true)
-            .build(), Child.builder()
             .firstName("Sarah")
             .lastName("Reeves")
             .gender("FEMALE")
             .dob("2-FEB-2002")
-            .isParty(true)
+            .isParty(false)
+            .build(), Child.builder()
```

Comment:
not formatted correct, please move  Child.builder() to new line and reformat to have have proper indentation